namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-contacts-dev/resources/domain-events-topic.tf

```diff
@@ -0,0 +1,14 @@
+resource "kubernetes_secret" "hmpps_contacts_hmpps_domain_events_topic" {
+  metadata {
+    name      = "hmpps-domain-events-topic"
+    namespace = var.namespace
+  }
+
+  data = {
```

Comment:
Remove this from initial build too - we can integrate with domain events topic later.

Reply:
On second thoughts - please leave it in - saves us a job later.